package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "docs": "scripts/markdown.js",
     "format": "npm-run-all format:*",
-    "format:pkg": "./lib/cli/index.js -wp",
+    "format:pkg": "./lib/cli/index.js -w package.json",
```

Comment:
Need to format only the root `package.json` and ignore the ones in `examples/*`.